docker-compose.yml

```diff
@@ -5,16 +5,12 @@ services:
     environment:
       MYSQL_ALLOW_EMPTY_PASSWORD: 'true'
       MYSQL_ROOT_HOST: '%'
-    volumes:
-      - ./docker/mysql/data:/var/lib/mysql
     ports:
       - '3306:3306'
   postgres:
     image: postgres:12
     environment:
       POSTGRES_USER: postgres
       POSTGRES_HOST_AUTH_METHOD: trust
-    volumes:
-      - ./docker/postgresql/data:/var/lib/postgresql/data
```

Comment:
Why did you remove it?

Reply:
`go get -t ./...` fails in github actions
https://github.com/k0kubun/sqldef/runs/2302532462?check_suite_focus=true